include/mapnik/svg/svg_path_attributes.hpp

```diff
@@ -135,7 +153,11 @@ struct path_attributes
           visibility_flag(attr.visibility_flag),
           display_flag(attr.display_flag),
           dash(attr.dash),
-          dash_offset(attr.dash_offset)
+          dash_offset(attr.dash_offset),
+          cached_images(attr.cached_images)
```

Comment:
so, you're adding the cache, offset and mutex to all attributes but just using it for the first one (`attrs[0]` below), right?

Reply:
Yes